### PR TITLE
Gif picker deprecation notice

### PR DESCRIPTION
**Pull request by marianunez**
#### Summary
Gfycat deprecation notice as defined in [here](https://docs.google.com/document/d/1Hryg5tEGnEDhIHJuW-agreIw6Bb7Ghoh43IF7fb8oNA/edit#bookmark=id.8xeqmx3dc5yi)

#### Jira ticket
https://mattermost.atlassian.net/browse/MM-54280

#### Test steps and expectation (required)

- Notice should appear for on-prem servers only: 
    - Servers below v7.8.9
    - v7.9-v7.10
    - v8.0.0 and v8.0.1
     - v8.1.0
- Notice should not appear in Cloud and should not appear in versions v.7.8.9, 8.0.2, 8.1.1 or 9.0 and above
- Notice should only appear for admins.
- Learn more button should go to the forum post.

### CODE REVIEW

File: notices.json [hunk 1]
```diff
@@ -1,4 +1,21 @@
 [
+  {
+    "id": "gfycat_deprecation",
+    "conditions": {
+      "audience": "sysadmin",
+      "instanceType": "onprem",
+      "serverVersion": ["<7.8.9", "8.0.0-8.0.1", "8.1.0"]
```
Comment (marianunez) [inline notices.json:7]: @johndavidlugtu @amyblais can you double check these server versions that the notice will be displayed:

- Servers below v7.8.9
- v8.0.0 and v8.0.1
- v8.1.0

All other versions after v8.1.0 shouldn't get the notice. Did I miss any?

Comment (amyblais) [reply notices.json:7]: Also v7.9 and v7.10 versions should get the notice.

Comment (marianunez) [reply notices.json]: Ah good catch. Missed those. I add the `"7.9-7.10"` which I'm hoping the notices is picked up by any dot releases on those versions

**Review by amyblais — APPROVED** [review]

**Review by yasserfaraazkhan — APPROVED** [review]
Hi @marianunez , 
just curious if this change should be going into `release` branch ?

once merged to master, I can test this on different server versions.

**Comment by marianunez** [comment]
> Hi @marianunez , 
> just curious if this change should be going into `release` branch ?
> 
> once merged to master, I can test this on different server versions.

Yes, we want to test this so I'll merge to `master` now so you can test @yasserfaraazkhan

**Comment by yasserfaraazkhan** [comment]
Sure @marianunez , I'll test it on the mentioned versions.

**Pull request merged**